adoc-mode.el

```diff
@@ -1556,6 +1556,19 @@ Subgroups of returned regexp:
 
 
 ;;;; font lock keywords
+(defsubst adoc-kwf-search (regexp &optional bound noerror count)
+  "Keyword search for Adoc.
+Like `re-search-forward' with the same arguments
+REGEXP, BOUND, NOERROR and COUNT.
+If a match for REGEXP is found where the text property
+`adoc-code-block' is non-nil continue the search."
```

Comment:
Probably you should also mention explicitly that the purpose of this is skip code in code blocks for performance reasons.

Reply:
Not only for performance reasons but also to prevent the adoc fontification within code blocks.

Reply:
Well, this should be mentioned in the docstring as well then.